### PR TITLE
[TorchBench] add performance github bot

### PR DESCRIPTION
**Pull request by zzpmiracle**

### CODE REVIEW

File: pytorch_blade/benchmark/TorchBench/results_anaysis.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+detail data can be seen in {oss_dir}
+created by TorchBench CI automatically
+"""
+
+
+def try_cast_to_float(to_cast: str):
+    try:
+        to_cast = float(to_cast)
+    except:
+        pass
```
Comment (tanyokwok) [inline pytorch_blade/benchmark/TorchBench/results_anaysis.py:38]: Nit: print the exception error message.

File: pytorch_blade/benchmark/TorchBench/results_anaysis.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import os
+import pandas as pd
+
+COMPARE_FIELDS = [
+    "disc (latency)",
+    "blade (latency)",
+    "dynamo-blade (latency)",
+    "dynamo-disc (latency)",
+]
+RELATED_DIFF_PERCENT = 3
```
Comment (tanyokwok) [inline pytorch_blade/benchmark/TorchBench/results_anaysis.py:22]: Nit: How about making the `RELATED_DIFF_PERCENT` configurable in CI.

File: pytorch_blade/benchmark/TorchBench/test_torch_bench.sh [hunk 3]
```diff
@@ -30,32 +30,43 @@ python3 -m pip install -U numpy
...
 
+# results
+date_str=$(date '+%Y%m%d-%H')
+oss_link=https://bladedisc-ci.oss-cn-hongkong.aliyuncs.com
+oss_dir=oss://bladedisc-ci/TorchBench/${bench_target}/${date_str}
 
+for result in ${results[@]}
+do
+    cat ${result}/summary.csv
+    curl ${oss_link}/TorchBench/baseline/${result}_${bench_target}.csv -o $result.csv
```
Comment (tanyokwok) [inline pytorch_blade/benchmark/TorchBench/test_torch_bench.sh:60]: Will the baseline be updated manually?

File: pytorch_blade/benchmark/TorchBench/results_anaysis.py (hunk 1 above)
Comment (zzpmiracle) [reply pytorch_blade/benchmark/TorchBench/results_anaysis.py:37]: some models cannot run currently, so it's rational that some results are like 'RuntimeError'， ’UnserializableException‘， we have handled it if status changed.

File: pytorch_blade/benchmark/TorchBench/results_anaysis.py (hunk 2 above)
Comment (zzpmiracle) [reply pytorch_blade/benchmark/TorchBench/results_anaysis.py]: Done.

File: pytorch_blade/benchmark/TorchBench/results_anaysis.py (hunk 1 above)
Comment (tanyokwok) [reply pytorch_blade/benchmark/TorchBench/results_anaysis.py:37]: OK.

File: pytorch_blade/benchmark/TorchBench/test_torch_bench.sh (hunk 3 above)
Comment (zzpmiracle) [reply pytorch_blade/benchmark/TorchBench/test_torch_bench.sh:60]: Yes, I will update it when some performance promoted.

**Review by tanyokwok — APPROVED** [review]
LGTM

**Pull request merged**